empress/support_files/js/empress.js

```diff
@@ -158,12 +158,26 @@ define([
             this._treeData[i].splice(this._tdToInd.visible, 0, true);
         }
 
+        /**
+         * @type {String}
+         * Text to display at the bottom of the continuous legend when some
+         *  values in a continuous are either missing or non-numeric.
+         */
+        this._continuousMissingNonNumericWarning =
+            "Some value(s) in this field were missing and/or not numeric. " +
+            "These value(s) have been left out of the gradient, and the " +
+            "corresponding nodes have been set to the default color.";
         /**
          * @type {Legend}
          * Legend describing the way the tree is colored.
          * @private
          */
         this._legend = new Legend(document.getElementById("legend-main"));
+        this._legend.setMissingNonNumericWarning(
+            "Some value(s) in this field were missing and/or not numeric. " +
+                "These value(s) are not included in the gradient, and the " +
+                "associated nodes have been left as the default color."
+        );
```

Comment:
```suggestion
            "Some value(s) in this field were missing and/or not numeric. " +
                "These value(s) are not included in the gradient, and the " +
                "associated nodes have been left as the default color.",
            "Missing / non-numeric value(s)' associated nodes left as " +
                "default color."
        );
```

Currently, no short message is set for the main legend, which messes with the legend SVG export:

![legs2](https://user-images.githubusercontent.com/4177727/121762287-cbc7ed00-cae9-11eb-909b-2d0cdd20ff44.png)
_top = main legend; bottom = barplot legend (both using modified data with some missing/non-numeric values)_

This fixes that.